homeassistant/components/isy994/entity.py

```diff
@@ -105,7 +110,9 @@ def device_info(self) -> DeviceInfo | None:
 
         if hasattr(node, "address"):
             assert isinstance(node.address, str)
-            device_info[ATTR_NAME] = f"{basename} ({node.address})"
+            device_info[
+                ATTR_NAME
+            ] = f"{basename} ({(node.address.rpartition(' ')[0] or node.address)})"
         if hasattr(node, "primary_node"):
```

Comment:
Side note: The device info identifiers or connections are not optional. Withe of them need to be present. This code looks like it makes the identifiers optional (maybe not in practice).

Reply:
Good catch, this PR fixes the ISY group entities, but I'll have to do another follow up to cover variable and program entities which don't have the primary_node attribute and also don't currently get a device assigned.

In practice, the identifiers will always exist for nodes or groups.